ArcGISRuntimeSDKQt_CppSamples/Layers/BrowseWfsLayers/BrowseWfsLayers.cpp

```diff
@@ -0,0 +1,169 @@
+// limitations under the License.
+// [Legal]
+
+#include "BrowseWfsLayers.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "WfsFeatureTable.h"
+#include "FeatureLayer.h"
+#include "QueryParameters.h"
+#include "SimpleRenderer.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleMarkerSymbol.h"
+#include "WfsService.h"
+#include "WfsLayerInfo.h"
+
+using namespace Esri::ArcGISRuntime;
+
+BrowseWfsLayers::BrowseWfsLayers(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_map(new Map(Basemap::topographic(this), this))
+{
+  // create WFS Service
+  m_wfsService = new WfsService(QUrl("https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities"),this);
+
+  // once WFS Service is loaded, populate QStringList with layer names for ComboBox
+  connect(m_wfsService, &WfsService::doneLoading, this, [this](Error e)
+  {
+    if (!e.isEmpty())
+      return;
+
+      m_wfsLayersInfoList = m_wfsService->serviceInfo().layerInfos();
+      for(auto i : m_wfsLayersInfoList)
+        m_layerInfoTitleList.append(i.title());
+
+      emit layerInfoTitleListChanged();
+  });
+  m_wfsService->load();
+}
+
+BrowseWfsLayers::~BrowseWfsLayers() = default;
+
+void BrowseWfsLayers::init()
+{
+  // Register the map view for QML
+  qmlRegisterType<MapQuickView>("Esri.Samples", 1, 0, "MapView");
+  qmlRegisterType<BrowseWfsLayers>("Esri.Samples", 1, 0, "BrowseWfsLayersSample");
+}
+
+MapQuickView* BrowseWfsLayers::mapView() const
+{
+  return m_mapView;
+}
+
+// Set the view (created in QML)
+void BrowseWfsLayers::setMapView(MapQuickView* mapView)
+{
+  if (!mapView || mapView == m_mapView)
+    return;
+
+  m_mapView = mapView;
+  m_mapView->setMap(m_map);
+
+  connect(m_mapView, &MapQuickView::navigatingChanged, this, [this]()
+  {
+    if (m_mapView->isNavigating())
+      return;
+
+    populateWfsFeatureTable();
+  });
+
+  emit mapViewChanged();
+}
+
+void BrowseWfsLayers::createWfsFeatureTable(int index, bool swap)
+{
+  // clear previous layer
+  m_map->operationalLayers()->clear();
+  // set viewpoint to extent of selected layer
+  m_mapView->setViewpointGeometry(m_wfsLayersInfoList[index].extent());
+  // create WFS Feature Table from selected layer
+  m_wfsFeatureTable = new WfsFeatureTable(m_wfsLayersInfoList[index], this);
+
+  // Set feature request mode to manual - only manual is supported at v100.5.
+  // In this mode, you must manually populate the table - panning and zooming
+  // won't request features automatically.
+  m_wfsFeatureTable->setFeatureRequestMode(FeatureRequestMode::ManualCache);
+
+  if(swap == true)
+    m_wfsFeatureTable->setAxisOrder(OgcAxisOrder::Swap);
+  else
+    m_wfsFeatureTable->setAxisOrder(OgcAxisOrder::NoSwap);
+
+  // once WFS Feature Table is loaded, populate the table and add the layer to the map
+  connect(m_wfsFeatureTable, &WfsFeatureTable::doneLoading, this, [this](Error e){
```

Comment:
newline for curly bracket. Either needs aligned with the `connect` or with the start of the capture `[this]`.